numpy/core/src/multiarray/temp_elide.c

```diff
@@ -7,6 +7,10 @@
 #include "npy_config.h"
 #include "numpy/arrayobject.h"
 
+#if defined HAVE_BACKTRACE
+#include <execinfo.h>
+#endif
```

Comment:
In other places, we use:
```
#if defined HAVE_EXECINFO_H
#include <execinfo.h>
#elif defined HAVE_LIBUNWIND_H
#include <libunwind.h>
#endif
```
See gh-22152.  That might be what is needed for Emscripten (or at least for something else).

Reply:
Ah yes, thanks!